projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -563,7 +563,7 @@ public static function parse( $attributes, $content, $context = array() ) {
 		wp_interactivity_config( 'jetpack/form', $config );
 		\wp_enqueue_script_module(
 			'jp-forms-view',
-			plugins_url( '../../dist/modules/form/view.js', __FILE__ ),
+			plugins_url( 'dist/modules/form/view.js', dirname( __DIR__ ) ),
```

Comment:
I updated this here: 
- https://github.com/Automattic/jetpack/pull/44542

You'll just need a rebase.

Reply:
rebased!